integration_test/config/base.yml

```diff
@@ -8,8 +8,6 @@ specs: [simple_nw,
         router_v2v_tp,
         wanedge,
         wanedge_tp,
-        promiscuous_seg_ovs,
-        promiscuous_seg_of,
```

Comment:
Since The promiscuous mode feature is still in OpenVNet, we can't just remove the tests. Yes, it's annoying that they're currently failing often on Centos 7 but let's not sweep the problem under the rug by just removing the tests. Especially since they're succeeding on Centos 6. Instead let's fix the feature so it starts working on Centos 7 too.

Reply:
Those tests were removed because the feature was incomplete and designed wrong, and only seemed to worked on centos 6 due to some quirks of the os.

It would need to be completely rewritten to make it work correctly.

Reply:
I see. Guess we'll just accept promiscuous mode as broken right now until the rewrite happens.